### PR TITLE
CSS for new pinned events indicator

### PR DESCRIPTION
**Pull request by turt2live**
see https://github.com/matrix-org/matrix-react-sdk/pull/1486

### CODE REVIEW

File: src/skins/vector/css/matrix-react-sdk/views/rooms/_RoomHeader.scss [hunk 1]
```diff
@@ -224,3 +224,17 @@ limitations under the License.
...
+
+.mx_RoomHeader_unreadPinsIndicator {
+    position: absolute;
+    right: 0;
+    bottom: 4px;
+    width: 8px;
+    height: 8px;
+    border-radius: 8px;
+    background-color: $warning-color;
+}
```
Comment (lukebarnard1) [inline src/skins/vector/css/matrix-react-sdk/views/rooms/_RoomHeader.scss]: No NL @ EOL

Comment (turt2live) [reply src/skins/vector/css/matrix-react-sdk/views/rooms/_RoomHeader.scss]: fixed

**Comment by ara4n** [comment]
hm, if these were approved 16 days ago, how come we didn't merge them?

**Comment by ara4n** [comment]
oh, because the react side isn't read. "hum".

**Pull request merged**